app/src/main/res/layout/activity_zim_host.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/constraintLayoutId"
```

Comment:
`@+id/zimHostRootLayout` if this needs an id

Reply:
I don't think this needs an Id. So fixed. 883abbcef91eec1ae2359d4753fc06d9ac828460